_episodes/05-writing-scripts.md

```diff
@@ -17,17 +17,23 @@ keypoints:
 
 We've been able to do a lot of work with files that already exist, but what if we want to write our own files. Obviously, we're not going to type in a FASTA file, but we'll see as we go through other tutorials, there area lot of reasons we'll want to write a file, or edit an existing file.
 
-To write in files, we're going to use the program `nano`. We're going to create a file to take notes on what we've been doing with the data files in **/dc_sample_data/untrimmed_fastq**.
+To write in files, we're going to use the program `nano`. We're going to create a file to take notes on what we've been doing with the data files in `/dc_sample_data/untrimmed_fastq`.
```

Comment:
`/dc_sample_data/untrimmed_fastq` should be `~/dc_sample_data/untrimmed_fastq`?

Reply:
I believe @swang8 is correct.